src/main/java/org/apache/sling/discovery/base/connectors/ping/package-info.java

```diff
@@ -23,8 +23,7 @@
  *
  * @version 1.0.0
  */
-@Version("1.0.0")
+@Version("2.0.0")
 package org.apache.sling.discovery.base.connectors.ping;
 
-import aQute.bnd.annotation.Version;
-
+import org.osgi.annotation.versioning.Version;
```

Comment:
Switching annotations (from Bnd to OSGi only requires a micro(!) version bump, i.e. 2.0.1 is totally fine.